File: packages/js/core/src/manifest/formats/polywrap.plugin/migrators/0.0.1-prealpha.2_to_0.0.1-prealpha.3.ts
/* eslint-disable @typescript-eslint/naming-convention */

import { PluginManifest as OldManifest } from "../0.0.1-prealpha.2";
import { PluginManifest as NewManifest } from "../0.0.1-prealpha.3";

export function migrate(_: OldManifest): NewManifest {
  throw new Error(
    "Plugin manifest file is deprecated. Please update to 0.0.1-prealpha.3"
  );
}

File: packages/py/manifests/polywrap-manifest/polywrap_manifest/formats/polywrap.plugin/migrators/_0_0_1_prealpha_2_to__0_0_1_prealpha_3.py
from ... import import_manifest
from typing import TypeAlias
import os

local_path: os.PathLike = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))

__0_0_1_prealpha_2 = import_manifest('0.0.2-prealpha.1', local_path)
OldManifest: TypeAlias = __0_0_1_prealpha_2.PluginManifest
__0_0_1_prealpha_3 = import_manifest('0.0.1-prealpha.3', local_path)
NewManifest: TypeAlias = __0_0_1_prealpha_3.PluginManifest


def migrate(_: OldManifest) -> NewManifest:
    raise Exception('Plugin manifest file is deprecated. Please update to 0.0.1-prealpha.3')
